internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -1121,8 +1121,9 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Default:  string(managedclusters.ManagedClusterSKUTierFree),
 				ValidateFunc: validation.StringInSlice([]string{
 					string(managedclusters.ManagedClusterSKUTierFree),
-					string(managedclusters.ManagedClusterSKUTierPaid),
 					string(managedclusters.ManagedClusterSKUTierStandard),
+					// Allow this value for the time being so users can be informed by the breaking change error message from the AKS API
+					"Paid",
```

Comment:
Rather than supporting this as a value and having users find out this is no longer supported when they try and update their cluster - since we're going to need to message this breaking change out anyway, we should remove this so that `terraform plan` can catch this:

```suggestion
```

That'd mean that users need to update their config, but since this is a breaking change in the API (and the value needs to be updated there) there's kinda not a lot we can do about it tbh